### DISCUSSION TITLE
Can’t install app I paid for

### DISCUSSION THREAD

**Discussion by wkorbin** [Kemmey/Kemmey-TeslaWatch-Public#548] [General]
I just paid the money to install the app on my phone and I keep getting an access denied error so I don’t know what to do to get it loaded

this is what I see. Please help. 

Access Denied

You don't have permission to access "http://auth.tesla.com/oauth2/v3/authorize?" on this server.
Reference #18.87cf5868.1676857130.2fa715d4

**Comment by Kemmey** [comment]
Hey 

Turn off any vpn anonymizing services during login. Tesla does not allow authenticating while using such services. This includes iCloud private relay. If on corporate WiFi with ssl inspection, this counts as well. 

Try turning off WiFi during login - or the reverse, if your telco is the one doing the packet inspection.  

Cheers,

Kim

**Comment by wkorbin** [comment]
hi Kim

 

ok thx. how do I get the app to appear on my watch?

 

TY!
